tools/mo/unit_tests/mo/utils/freeze_placeholder_test.py

```diff
@@ -189,35 +192,50 @@ def test_freeze_placeholder_with_value_onnx_fe(self, input_freezing_value, use_n
                 np.array([-2.0, -2.0, -2.0]),
                 np.float32,
             ),
+            (
+                    "in1",
+                    "in2->[-1.0]",
+                    True,
+                    {
+                        "in1": np.array([2.0, 2.0, 2.0]).reshape(1, 1, 3)
+                    },
+                    np.array([-2.0, -2.0, -2.0]),
+                    np.float32,
+            ),
             (
                 "in1[3 1]{f32}->[7.0 11.0 -1.0],in2{f32}->3.0",
+                None,
                 True,
                 {},
                 np.array([21.0, 33.0, -3.0]).reshape(3, 1),
                 np.float32,
             ),
             (
                 "in1[3 1]{f16}->[7.0 11.0 -1.0],in2{f16}->3.0",
+                None,
                 True,
                 {},
                 np.array([21.0, 33.0, -3.0]).reshape(3, 1),
                 np.float16,
             ),
             (
                 "in1[3 1]{i32}->[7 11 -1],in2{i32}->3.0",
+                None,
                 True,
                 {},
                 np.array([21, 33, -3]).reshape(3, 1),
                 np.int32,
             ),
         ],
     )
-    def test_freeze_placeholder_with_value_mul(self, input_freezing_value, use_new_fe, inputs, expected, dtype=None):
+    def test_freeze_placeholder_with_value_mul(self, input_freezing_value, freeze_placeholder_value,
```

Comment:
Can you add tests for freezing with not only FP32 value, but also bool and int? Or is it not supported?

Reply:
Done.